### DISCUSSION TITLE
Does SQlite Beta X Version write more to disk than the previous stable version 0.56

### DISCUSSION THREAD

**Discussion by tweek11** [mealie-recipes/mealie#1833] [General]
Please pardon my ignorance regarding this question - Does SQlite Beta X Version write more to disk than the previous stable version 0.56?

The reason i ask is i run Proxmox with ZFS on SSD drives with a Ubuntu Server 22.04 LTS VM - I'm in the process of building out another Proxmox Server with ZFS on larger spinning drives for backups and more write intensive VM's.

An if ZFS is not watched it can easily burn through SSD's via constant writes running in the back ground.

Hence the question above if SQLite writes more (current stable version is perfectly fine on the SSD VM) - If so i'll create another VM on the newer server for the SQLite beta version with spinning drives.

Also in the SQLite docker-compose.yaml example it show the following
 deploy:
      resources:
        limits:
          memory: 1000M
since this is only being access by one or two people is 1000M the minimal suggestions to run? An is it safe to assume that means 1 gig of memory. Looking for clarification on minimal memory setup for essentially one or two people - Didnt know if more memory means less writes or less memory means more writes when not being accessed and used. 

Last thank you and the others for all your work on such a awesome project. Absolutely love it over the others i have tried. Thank you again.

**Comment by michael-genson** [comment]
Mealie has changed drastically since this discussion was opened, and resource consumption has changed, including many optimizations.

> is 1000M the minimal suggestions to run?

I think this is just a safe default, your best bet is to try reducing it and seeing if performance is sufficient or not.
